stock_picking_report_product_sticker/tests/common.py

```diff
@@ -0,0 +1,46 @@
+from odoo.tests import tagged
+
+from odoo.addons.product_sticker.tests.common import ProductStickerCommon
+
+
+@tagged("post_install", "-at_install")
+class ProductStickerStockCommon(ProductStickerCommon):
+    @classmethod
+    def setUpClass(cls):
+        super().setUpClass()
+        imd_model = cls.env["ir.model.data"]
+        sl_model = cls.env["stock.location"]
+        cls.picking_type_out = cls.env["stock.picking.type"].browse(
+            imd_model._xmlid_to_res_id("stock.picking_type_out")
+        )
+        cls.picking_type_out.show_product_stickers = True
+        cls.stock_location = sl_model.browse(
+            imd_model._xmlid_to_res_id("stock.stock_location_stock")
+        )
+        cls.supplier_location = sl_model.browse(
+            imd_model._xmlid_to_res_id("stock.stock_location_suppliers")
+        )
```

Comment:
Let's clean this up a bit:

```suggestion
        cls.picking_type_out = cls.env.ref("stock.picking_type_out")
        cls.picking_type_out.show_product_stickers = True
        cls.stock_location = cls.env.ref("stock.stock_location_stock")
        cls.supplier_location = cls.env.ref("stock.stock_location_suppliers")
```